siriuspy/siriuspy/clientconfigdb/types/as_rf.py

```diff
@@ -719,7 +753,7 @@ def get_dict():
 
 _template_dict = {
     'pvs':
-    _pvs_as_rf +
-    _pvs_li_llrf + _pvs_bo_llrf + _pvs_bo_rfssa + _pvs_bo_rfcal +
+    _pvs_as_rf + _pvs_li_llrf + 
+    _pvs_bo_pow_sensor + _pvs_bo_llrf + _pvs_bo_rfssa + _pvs_bo_rfcal +
     _pvs_si_llrf + _pvs_si_rfssa + _pvs_si_rfcav + _pvs_si_rfcal
-    }
+    }
```

Comment:
missing new line at the end of the file.

Reply:
Thanks for noticing that! I’ll correct that immediately